Makefile

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-#  Copyright 2019 The Kubernetes Authors.
+#  Copyright 2020 The Kubernetes Authors.
```

Comment:
more for my own knowledge, do we usually update copyright year in kubernetes when a file is updated?

Reply:
I don't have a good answer, I changed it because it was mostly rewritten